src/Editor.ts

```diff
@@ -227,17 +227,37 @@ export default class Editor extends EventDispatcher {
     return this.set(new TextDocument(new Delta().insert(text), selection), source);
   }
 
-  getActive() {
-    const { selection } = this.doc;
-    let active = selection
-      ? selection[0] === selection[1]
-      ? { ...this.activeFormats, ...this.doc.getLineFormat(selection) }
-      : { ...this.doc.getFormats(selection) }
-      : {};
-    Object.values(this.modules).forEach(module => {
-      if (module.getActive) active = { ...active, ...module.getActive() };
-    });
-    return active;
+  fixupSelection(selection) {
```

Comment:
Be sure to add types. `selection: EditorRange`

Reply:
Perhaps a better name would be helpful. `trimSelection`?

Reply:
Did those two things.